src/components/ProvinceDropdown/province.data.js

```diff
@@ -0,0 +1,392 @@
+export default [
+  {
+    code: "TH-10",
+    title_th: "กรุงเทพมหานคร",
+    type: "เขตปกครองพิเศษ",
+  },
+  {
+    code: "TH-11",
+    title_th: "สมุทรปราการ",
+    type: "จังหวัด",
+  },
+  {
+    code: "TH-12",
+    title_th: "นนทบุรี",
+    type: "จังหวัด",
```

Comment:
Also, since most fields have `type: "จังหวัด"` in here, I personally like to use `map` to shorten the data a bit.

```
provinces.map(province => ({type: "จังหวัด", ...province}))
```

Then, you can omit `type: "จังหวัด"` in most of the items.